upstairs/src/lib.rs

```diff
@@ -81,6 +81,10 @@ async fn process_message(
         Message::Imok => Ok(()),
         Message::WriteAck(uuid, ds_id, result) => {
             if u.uuid != *uuid {
+                println!(
```

Comment:
When would we get this situation?  It seems like it would only happen if a downstairs sent us a response after that downstairs had switched to a different upstairs?  Or do I have it wrong there.

I know all you did was add a prinln! here, but I'm now curious about the situation where we would see these.

Reply:
Should we panic then?  If we just print a message, I'll never see it.  If it panics the downstairs then there is a chance I'll see it :)